### ISSUE TITLE
Autoformat not working anymore with stylelint-config-standard 18.0.0

### ISSUE THREAD

**Issue by jimblue** [mrmlnc/vscode-stylefmt#138]
### Environment

  * VSCode Version: 1.18.1
  * OS Version: OSX 10.13.1

### Actual behavior
Not formatting on save or with command

### Expected behavior
Should format (was working before upgrading to stylelint-config-standard 18.0.0)

### Steps to reproduce
In `.stylelintrc`
```json
"extends": "stylelint-config-standard",
  "plugins": [
    "stylelint-scss",
    "stylelint-order"
  ]
...
```

### Reference
https://github.com/stylelint/stylelint-config-standard/issues/99

**Comment by mrmlnc** [comment]
Related to https://github.com/morishitter/stylefmt/issues/327.

**Comment by jimblue** [comment]
@mrmlnc I don't think this issue it's not related to the one you point to as I don't get this error since I've added the following in `.stylelintrc`:
`"no-descending-specificity": null`

**reference**
https://github.com/stylelint/stylelint-config-standard/issues/98

**Comment by jimblue** [comment]
It's just not formatting anymore... I just tried rollback to `stylelint-config-standard 17.0.0` and it fix the problem.
So I guess there is some kind of compatibility issu between `stylefmt` and `stylelint-config-standard 18.0.0`

**Comment by jimblue** [comment]
@mrmlnc are you able to reproduce this problem?

**Comment by jimblue** [comment]
Ok apparently the problem come from `stylefmt` because there not up to date...

https://github.com/stylelint/stylelint/issues/3042#issuecomment-347578941

I've notice the issue, hopefully they'll correct it soon enough 😄 

Thanks for your rapid answer

**Comment by smlombardi** [comment]
It no longer formats scss files, even on command.

Is this plugin abandoned, can someone recommend an alternative?

**Comment by mrmlnc** [comment]
Yeap, you can use Stylefmt as CLI command. Also you can create issue for your problem or send PR to my repository or `stylefmt` to fix this problem (https://github.com/morishitter/stylefmt/issues/328).

**Comment by smlombardi** [comment]
OK, I updated the dependency in the package.json and did a PR.

**Comment by smlombardi** [comment]
FWIW, I switched to Prettier and https://github.com/hugomrdias/prettier-stylelint

It not only formats the code but orders the attributes.

**Comment by jimblue** [comment]
Also switch to prettier +1

**Comment by mrmlnc** [comment]
Fixed by #162.
